Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml.cs

```diff
@@ -181,6 +188,27 @@ async void RunMBaaSAsync(object sender, EventArgs e)
         void SignOut(object sender, EventArgs e)
         {
             Auth.SignOut();
+            userInfo = null;
+            SignInInformationButton.Text = "User not authenticated";
```

Comment:
A constant?

Reply:
Might have been made one, yeah. Didn't make one since other code don't tend to use constants for strings.
Guess it should be fixed